From ce5f4ce490d70761ceb86d59fdac65c39b90aa1a Mon Sep 17 00:00:00 2001
From: Adrian Stratulat <adrian.stratulat91@gmail.com>
Date: Tue, 2 Jun 2020 23:13:18 +0300
Subject: [PATCH] Add shmem dependency to jt9

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e951b52c..b3f450bee 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -249,6 +249,7 @@ set (wsjt_qtmm_CXXSRCS
   )
 
 set (jt9_FSRCS
+  lib/shmem.cpp
   lib/jt9.f90
   lib/jt9a.f90
   )
